app/store/ephemeral_store.ts

```diff
@@ -3,6 +3,10 @@
 
 import {BehaviorSubject} from 'rxjs';
 
+import {toMilliseconds} from '@utils/datetime';
+
+const TIME_TO_CLEAR_WEBSOCKET_ACTIONS = toMilliseconds({seconds: 30});
```

Comment:
I guess this is enough time but still sensitive to race conditions, isn't it? 0/5 if we want to give it a try before changing to a different approach

Reply:
TL;DR: I tested on community and is more than enough time, but happy to pump up the numbers.

In my particular case (tested on community) we are talking about less than a second. I pumped those numbers to 30 seconds thinking on really slow networks. For what is worth, we could keep those as far as you want. I don't expect them to be neither clashing with other executions (once a post with id X is created, there is no way a new "Create -> Edit" path happens again for the same post), not grow too fast in a short time (it is only stored for edit and delete events that happen on posts not in the database).

The only time I could foresee this having some issue would be if you wrangle an old thread (not in the database because it is old) that has many messages. Many "post deleted" events may come for non existing posts, populating the structures. But even in the case that becomes a problem, I don't think the problem is really about the time those value live in memory.